project-minimization-core/src/main/kotlin/org/plan/research/minimization/core/algorithm/graph/hierarchical/LayerToCutTransformer.kt

```diff
@@ -0,0 +1,11 @@
+package org.plan.research.minimization.core.algorithm.graph.hierarchical
+
+import org.plan.research.minimization.core.model.DDContextWithLevel
+import org.plan.research.minimization.core.model.DDItem
+import org.plan.research.minimization.core.model.graph.Graph
+import org.plan.research.minimization.core.model.graph.GraphCut
+import org.plan.research.minimization.core.model.graph.GraphEdge
+
+interface LayerToCutTransformer<V, E, G, C> where V : DDItem, E : GraphEdge<V>, G : Graph<V, E, G>, C : DDContextWithLevel<C> {
```

Comment:
I'm sure that we can make one implementation, so we don't need the interface for this purpose

Reply:
It requires a lot of contextual information, so I'm not sure we need to implement it here. WDYT?